content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,85 @@
+# Kubernetes Removals and Deprecations In 1.24
+Thursday, March 31, 2022
+
+**Authors**: Mickey Boxell (Oracle), Debabrata Panigrahi (Independent)
+
+As Kubernetes evolves, features and APIs are periodically revisited and removed. New features may offer an alternative or improved approach to solving an existing problem, motivating the team to remove the old method. Rather than immediately being removed, old APIs are first deprecated and then removed according to the [Kubernetes Deprecation Policy](/docs/reference/using-api/deprecation-policy/). 
+
+We want to make sure you are aware of the changes coming in the Kubernetes 1.24 release. Kubernetes 1.24 is scheduled to be released in April and includes several removed and deprecated APIs. **Removed** APIs are those that you can use in current, supported Kubernetes versions but have already been deprecated. These removals have been superseded by newer, stable/generally available (GA) APIs. **Deprecated** APIs have been marked for removal in a future Kubernetes release. 
+
+For an exhaustive list of the changes coming in Kubernetes 1.24, take a look at the [CHANGELOG](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md). 
+
+## API removals and deprecations for Kubernetes 1.24
+
```

Comment:
I feel like the space between the H1 and H2 tags here are a great place to mention the Kubernetes Deprecation Policy. I'm getting the feeling the greater usersphere might be confused still by deprecated and removed. I know it seems straight forward in totality but a lot of folks were recently introduced to the concept (ie Dockershim deprecation/removal). https://kubernetes.io/docs/reference/using-api/deprecation-policy/

Reply:
I see a mention of it later in the doc. Pull it up here, some quick definitions of each, and done.